utils/check_repo.py

```diff
@@ -69,6 +69,7 @@
     "Pop2PianoStack",
     "SwitchTransformersStack",
     "TFDPRSpanPredictor",
+    "GroundingDinoTextModel",
```

Comment:
This is because it's an internal building block of the model, hence doesn't need to be in the main init or docs.

The reason we have to add it here is because the class inherits from `PretrainedModel` such that we can leverage the `get_extended_attention_mask` method.

Reply:
Yeap, and this is why the class was named `GroundingDinoTextPrenet` before as in the `check_repo.py` modules that end with `Prenet` are marked as private

Reply:
Done